js/templates/modals/settings/addressesForm.html

```diff
@@ -0,0 +1,83 @@
+<form class="box border padKids padStack clrS clrBr js-addAddressForm">
```

Comment:
I think when the first two tabs were created they were done off of now-outdated designs. I just copied the markup over from them and didn't retrofit the stylings with the thought that @morebrownies could do such tweaks.